ggshield/output/text/message.py

```diff
@@ -24,6 +25,35 @@
 MAX_SECRET_SIZE = 80
 
 
+def remediation_message(
+    remediation_steps: str, bypass_message: str, rewrite_git_history: bool = False
+) -> str:
+
+    rewrite_git_history_message = (
+        """
+  To prevent having to rewrite git history in the future, setup ggshield as a pre-commit hook:
+      https://docs.gitguardian.com/ggshield-docs/integrations/git-hooks/pre-commit
+
+"""
+        if rewrite_git_history
+        else ""
+    )
+
+    return """{} How to remediate
+
+{}
+{}
+{} [To apply with caution] If you want to bypass ggshield (false positive or other reason), run:
+{}
+    """.format(
+        format_text(">", STYLE["detector_line_start"]),
+        remediation_steps,
+        rewrite_git_history_message,
+        format_text(">", STYLE["detector_line_start"]),
```

Comment:
I would rework this to use f-strings. Something like this:

```python
line_start = format_text(">", STYLE["detector_line_start"])

return f"""{line_start} How to remediate

  {remediation_steps}
  {rewrite_git_history_message}
  {line_start} [To apply with caution] If you want to bypass ggshield (false positive or other reason), run:
  {bypass_message}
"""
```

Also by including the 2-spaces indent in this string, it can be removed from remediation_steps, rewrite_git_history_message and bypass_message.

Reply:
Ah right, that would mean only the first line would not be indented. Never mind then.